ImageGalery/GaleryTableViewController.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  GaleryTableViewController.swift
+//  ImageGalery
+//
+//  Created by apple on 18/07/2018.
+//  Copyright © 2018 wolfpack. All rights reserved.
+//
+
+import UIKit
+
+class GaleryTableViewController: UITableViewController, UICollectionViewDelegate, galleryUpdateDelegate {
+    
+    var galleryNames = [String]()
+    var activeGallerys = [ImageGallery]() {
+        didSet{
+            galleryNames.removeAll()
+            for index in activeGallerys.indices {
+                galleryNames.append(activeGallerys[index].title)
+            }
+        }
+    }
+    var recentlyDeletedGalery = [ImageGallery]()
+    var gallerys: [[ImageGallery]] {
+        return [activeGallerys, recentlyDeletedGalery]
+    }
+
+    
+    override func viewDidLoad() {
```

Comment:
you can remove this method if you only call super, for simplicity readability purpose